lib/grape-swagger/endpoint.rb

```diff
@@ -232,17 +232,26 @@ def expose_params_from_model(model)
       return model_name if @definitions.key?(model_name)
       @definitions[model_name] = nil
 
-      properties = {}
+      properties = nil
+      parser = nil
+
       GrapeSwagger.model_parsers.each do |klass, ancestor|
         next unless model.ancestors.map(&:to_s).include?(ancestor)
 
-        model_class = klass.new(model, self)
-        properties = model_class.call
+        parser = klass.new(model, self)
 
         break
       end
 
-      @definitions[model_name] = { type: 'object', properties: properties || {} }
+      properties = parser.call unless parser.nil?
+
+      if parser.nil?
+        properties = { unsupported_model_parser: { type: :string, description: "No parser registed for #{model_name}" } }
```

Comment:
If this is capitalized, it should be a full sentence, so ends with a period, please.